src/scripts/eastern_kingdoms/eastern_plaguelands/naxxramas/instance_naxxramas.cpp

```diff
@@ -751,7 +751,7 @@ bool instance_naxxramas::IsEncounterInProgress() const
 
 void instance_naxxramas::SetData(uint32 uiType, uint32 uiData)
 {
-    ASSERT(this)
+    ASSERT(this);
```

Comment:
How can this ever be not true?

Reply:
If SetData is called on a nullptr ScriptedInstance pointer.